usage-events.py

```diff
@@ -96,6 +96,29 @@
         }
     },
 ]
+context_menu = [
+    {
+        "id": "add-node",
+        "content": "Add Node (JS)",
+        "tooltipText": "Add Node",
+        "coreAsWell": "true",
```

Comment:
reminder: get rid of `coreAsWell` from this example -- if it's needed, just insert it for all items in the JS code